libs/datamodel/core/src/dml/mod.rs

```diff
@@ -1,20 +1,10 @@
-//! This module contains the models representing the Datamodel part of a Prisma schema.
-//! It contains the main data structures that the engines can build upon.
-mod datamodel;
-mod default_value;
-mod r#enum;
-mod field;
-mod model;
-mod relation_info;
-mod traits;
-
-pub use self::datamodel::*;
-pub use default_value::*;
-pub use field::*;
-pub use model::*;
-pub use r#enum::*;
-pub use relation_info::*;
-pub use traits::*;
-
-// Compatibility exports so that users of this module don't need to import the connector as well.
-pub use datamodel_connector::scalars::ScalarType;
+pub use dml::datamodel::*;
+pub use dml::default_value::*;
+pub use dml::field::*;
+pub use dml::model::*;
+pub use dml::native_type_constructor::*;
+pub use dml::r#enum::*;
+pub use dml::relation_info::*;
+pub use dml::scalars::*;
+pub use dml::traits::*;
+pub use dml::NativeTypeInstance;
```

Comment:
With the refactoring in the `lib.rs` I suggested this could become `pub use dml::*`.